### ISSUE TITLE
Positive acute response to NO challenge

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6508]
**Preferred term label:**
Positive acute response to NO challenge
**Synonyms**
**Definition (free text, please give PubMed ID)**
A reduction of mean pulmonary artery pressure (mPAP) ≥ 10 mmHg to reach an absolute value of mPAP ≤ 40 mmHg with an increased or unchanged cardiac output (CO) in response to inhaled nitric oxide at 10-20 ppm; PMID: 15939821
**Parent term (use hpo.jax.org/app)**
Pulmonary arterial hypertension HP:0002092
**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Idiopathic pulmonary arterial hypertension ORPHA:275766
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik 
Can we do this
label: Pulmonary arterial hypertension with positive acute response to NO challenge
definition: A form of pulmonary arterial hypertension in which there is a reduction of mean pulmonary artery pressure (mPAP) ≥ 10 mmHg to reach an absolute value of mPAP ≤ 40 mmHg with an increased or unchanged cardiac output (CO) in response to inhaled nitric oxide at 10-20 ppm; PMID: 15939821

**Comment by eswietlik** [comment]
That sounds great!

**Comment by pnrobinson** [comment]
added, thanks
